### PR TITLE
feat(content): Remnant: Relabelling Albatross as Medium Warship

### PR DESCRIPTION
**Pull request by Zitchas**
**Content**

## Summary
This changes the classification of the Remnant Albatross as a Medium Warship. No stats or attributes have been changed.

Both the design and utilization of the Albatross by the Remnant is more akin to that of the medium warship role, rather than the heavy warship (shield tank build aside). Likewise, it's gun-heavy design and focus on mobility are also more typical traits of medium warships rather than heavy ones. 

This change is basically intended to help contextualize and shape the perception of the Albatross to match the Remnant's view of it. 

No changes to statistics or attributes are planned.

## Save File
This save file can be used to play through the new mission content:
[Rebbia Rampart~3033-01-14 Remnant Capital License.txt](https://github.com/endless-sky/endless-sky/files/11345286/Rebbia.Rampart.3033-01-14.Remnant.Capital.License.txt)

## Artwork Checklist
 - ~~[X] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified~~
 - ~~[X] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: {{insert PR link}}~~
 - ~~[X] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}~~

### CODE REVIEW

**Comment by Amazinite** [comment]
We have heavy warships which are more relatively mobile (even considering how overly-mobile heavy warships in general are), such as the Falcon, Kestrel, or Derecho. We also have heavy warships which are gun-focused, such as the Vanguard, Shield Beetle, or Chigiriki. I'd say that the Shield Beetle even fits the "relatively mobile" bill. You say "shield tank build aside," but I'd say that if anything, being tanky and having lots of firepower, as the Albatross does, is part of what makes it solidly a heavy warship.

**Comment by Zitchas** [comment]
It's not "taking away" anything, it's merely assigning it a label that better fits how the Remnant view it. 

Also, basically every heavy warship you list are all ones that are called on a regular basis as showcasing why combat in ES is broken, and there is fundamentally no reason to fly anything other than the heaviest ship available.

In any case, I'll let the faction leads for those factions figure out what they would categorize their ships. For the Remnant, though, the Albatross is a medium warship. More properly speaking, the Albatross is a "Capital ship" equivalent to the Ibis, and should accordingly be found alongside it.

**Comment by warp-core** [comment]
> I'll let the faction leads for those factions figure out what they would categorize their ships.

Is the goal to create a collection of independent pieces of content from various different people who have something they want and don't really care about how it fits in with anything else or to create an actually cohesive game?

**Comment by DarcyManoel** [comment]
> More properly speaking, the Albatross is a "Capital ship" equivalent to the Ibis, and should accordingly be found alongside it.

I find myself agreeing with this more than my original comment, though I still believe that moving the Albatross down a warship category would further throw the ship categories out of wack. It would be nice to find ships categorised by their ship lines or manufacturer, rather than the arbitrary sorting that's causing debate for the *nth* time.

**Comment by AvianGeneticist** [comment]
A bit of a footnote, but there are already several jokes about the Met Par Tek 53 being the "Heaviest Light Warship" due to its colossal outfit space. I feel like relabelling the Albatross would essentially become that but for the Medium Warship category, due the Albatross' size, powerful integrated features, and durability.

There's also the issue of how many "fast and gun-focused" heavy warships there are. If we changed the Albatross, then (off the top of my head) the Falcon, (Vanguard kind of,) Shield Beetle (as Darcy said above), Derecho, and Chigriki should all be reclassed as well.

**Comment by Zitchas** [comment]
That's true: Our current "Heavy Warship" design and set are exemplars of how broken our combat system is by establishing that heavy warships are nothing more than extremely tough interceptors. Some are even *faster* than interceptors. Fixing the whole system is one of the goals of the Delta project.

In the meantime, though, labels are just that: Labels. They change nothing about the performance, attributes, or characteristics of a ship; and are merely there to convey to the player how they fit into their ecosystem. 

This is one of the reasons I am proposing to reclassify the Albatross there, because in terms of how they are used and how they fit into Remnant society and the Ember Waste, they are used as particularly heavy Medium Warships, not Heavy Warships.

As far as the labels themselves goes, the player is never restricted to a medium warship, (nor a light warship, for that matter). Nothing in the game depends on or emphasizes that the player should be sticking to a light or medium anywhere, so the fact that something is the "biggest light warship" or the "biggest medium warship" is an utterly irrelevant and meaningless distinction. Within any particular shipyard or faction, mediums are always better than lights, and heavies are always better than mediums. The player is always pushed (agressively) to get into a bigger ship, and the labels matter nought.

**Comment by EjoThims** [comment]
Albatross, to me, always felt like it had had just enough stuff added to barely push it into the lowest end of the HW threshold.

I think it would need to give up some combined HP at least, and probably 15-30 space (depending on how it's sliced) as well, and then it could comfortably be justified as one of the toughest/biggest WMs because as a MW it's still the largest starting Remnant combat ship.

Alternatively, it would make a lot of sense to me to go ahead and just completely refigure the Remnant Shipyard with custom categories instead. Not just because of the differences in design, manufacture, and usage paradigms but also social ones.

It would be especially neat to see the "living ship" line not only added with Merganser, but expanded with Peregrine. :D

**Comment by Zitchas** [comment]
> > More properly speaking, the Albatross is a "Capital ship" equivalent to the Ibis, and should accordingly be found alongside it.
> 
> I find myself agreeing with this more than my original comment, though I still believe that moving the Albatross down a warship category would further throw the ship categories out of wack. It would be nice to find ships categorised by their ship lines or manufacturer, rather than the arbitrary sorting that's causing debate for the _nth_ time.

I'd like this, but it also means we have to really think about what our shipyard is trying to do. Right now, it's built to facilitate the player easily finding the heavies available ship. Full stop, that's the whole point. It lets the player filter out all those pesky worthless lesser ships to gravitate it on the category of the biggest ship they can afford right now. 

Sort of extending beyond this, while it's always been a bit of an issue, it's become even more pronounced now that we have the Gegno and Bunrodea. Namely, when I look at the map shipyard, I've got half a million ships cluttering the list that I have zero access to from where I am. In reality, the only thing having such broad "interconnected" listings does is ensure that once someone gains access to another ecosystem of ships, they are thereafter actively discouraged from ever buying anything from the other shipyards ever again. Being able to compare ships is pretty convenient, but the gameplay impact is to devalue their lore, role, or story and put the focus on "bigger is better."

> Is the goal to create a collection of independent pieces of content from various different people who have something they want and don't really care about how it fits in with anything else or to create an actually cohesive game?

A fundamental, basic nature of the game is that we have a variety of storylines that aren't part of the "main" story. This is a fact. It's not going to change. Likewise, we have a whole range of ships from shipyards that have zero connection to humanity or to each other. This is also not something that is going to change. This game's very sandbox nature is that the player is going to gain access to a variety of ships in different places from cultures and societies that have significantly different values, development styles, and methodologies. This does mean that they are going to classify things differently. What is a heavy warship for one faction, might be equivalent to a freighter for another. Neither one is right or wrong.

What we have to decide is what is the goal: Are we trying to emphasize hp and aggressively push the player to get into the biggest ship possible? Or are we trying to craft a world with detail, variation, and interesting lore? Right now, this game puts a heavy emphasis on "bigger is better" and driving players aggressively into Heavy Warships. I don't think this is a good thing. In fact, I think this is a *bad* thing. Shifting focus to the lore and character of ships and how the societies that built them use them makes them more interesting, and better for the story. Yes, it does make it slightly harder to sort and compare ships easily, but we need better ways to do that anyway.

I take the view that ships are, effectively, the secondary protagonist. Where the ship fits into the story where it is being told matters massively more than how it fits into totally irrelevant other areas of the game. Ships can actually have character, differences, story, lore! Unlike our "protagonist" which has zero character. No description, no attributes, no skills, nothing. Our character is nothing more than two names, a bank account balance, and a combat rating. Oh yes, and the detail that they are a textile worker and a starship captain. Six points of data, only two of which we can directly control, and an additional 2 that we indirectly affect. Contrast with the Ship, where we have blocks of description, dozens of attributes, stories, lots of customization, and can even take actions and improve!

As such, I am *strongly* of the view that we should be classifying them by their role and usage in their society to further develop their character, since they are, in fact, actually major characters, rather than trying to nail them down in a "their hp relative to ships from some other part of the galaxy" system.

**tldr:** Categorization is purely a lore detail, a story element. As such, it should be selected based on how it contributes to the story and character of the ship.

**Comment by Quantumshark** [comment]
> Sort of extending beyond this, while it's always been a bit of an issue, it's become even more pronounced now that we have the Gegno and Bunrodea. Namely, when I look at the map shipyard, I've got half a million ships cluttering the list that I have zero access to from where I am.

This isn't really a related issue, but I would like it if there was an option on the map shipyard to exclude all ships that aren't sold somewhere you can land, and maybe a second option to exclude ships that require licenses you don't have. The same options could also be made available for outfits, which would also be useful. 

> As such, I am _strongly_ of the view that we should be classifying them by their role and usage in their society to further develop their character, since they are, in fact, actually major characters, rather than trying to nail them down in a "their hp relative to ships from some other part of the galaxy" system.

This part, I definitely agree with. I'd also much prefer ship categories based on ships' roles over the current ones which are solely based on size. But I think if we are making that change (and to reiterate, I do think that we should), then we should make it _everywhere_ rather than just on this one specific ship.

**Comment by Zitchas** [comment]
We've already started down that road with the additional categories, I just think we're much too focused on mass and hp.

It also brings up the engine problem again. Which ties into the mass problem. I really wish Amazinite had just merged their initial engine and mass rebalance. It wasn't perfect, but it was a big step in the right direction, and it'd make working on all this stuff a lot easier instead of having to sit here and wait for the theoretical future day when it'll all be handled all at once. 

In any case, while holistic change-everything-at-once solutions are nice in theory, I don't think they are practical. Which is why I'm just concerned with getting the Remnant ship classification aligned properly, rather than trying to change everything everywhere. I'm fairly sure that trying to change everything everywhere will *never* get any kind of widespread agreement.

**Comment by Zitchas** [comment]
Boiled down to the simplest level, the question here is what's more important:
- Where the Albatross appears in the map panel listing? or
- Where it appears in the actual shipyard where it is for sale?

Or alternatively, "Which is more important?"
- Where it is listed in the meta? or
- Where it is listed in the shipyard it is encountered in?

I think my fairly obvious opinion is that where it is displayed in the shipyard it is a part of is infinitely more valuable and important than where it is listed in the meta or in the "where to find this ship" list in the map panel.

**Comment by Zitchas** [comment]
map panel = meta, yes.

And yes, this is changing it in the shipyard.

I'm quite literally saying that where it appears in the shipyard is more important than where it appears in the meta or the map panel. And in the shipyard, it should be listed alongside the other Remnant capital ship, the Ibis.

**Comment by Zitchas** [comment]
Hasn't been decided yet, but I rather like the idea of just adding a "Living ship" category that the merganser, peregrine, and any future ships get assigned, and quite obviously if it has both merganser and peregrine in it, it's not being determined by mass or hp totals.

**Comment by Quantumshark** [comment]
If you're going for role-based classes, would it make sense to reclassify the Peregrine from a Medium Warship to a Light Warship? The Peregrine's extremely high speed already points towards light warship, and the cloaking suggests it'd be used in a similar manner to the Starling - cloaking is a massively powerful feature, and if anything is an even bigger distinction than the Peregrine's speed.

**Comment by Zitchas** [comment]
Probably would be rather reclassifying the starling, gull, penguin, and peregrine to "covert ships" or something.

**Comment by quyykk** [comment]
We probably shouldn't have a million categories for ships, it just makes it harder for players to remember which category a given ship is if they search for it, IMO

**Comment by Zitchas** [comment]
Hence why this just shifts it to the "Medium" category. Nice and simple, doesn't add any extra confusion to the game, and accomplishes the purpose. I'd prefer renaming the category, but one way or another, just having them next to each other in the same category is sufficient.

Although to be honest, I *really* doubt that anyone forgets where to buy Remnant ships. It's not exactly as if there's a lot of options...

**Comment by Amazinite** [comment]
Compared to the Ibis, the Albatross hull costs 74% more, has 127% more HP, 175% more required crew, 57% more outfit space, and 60% more weapon space. They're really not comparable ships.

**Comment by warp-core** [comment]
The arguments that have been made so far for the Albatross remaining in the "heavy warship" category refer to its stats and how these compare to other ships. Your arguments for it moving to the "medium warship" category make vague allusions to its "role," and yet I see no actual description of its "role," just the assertion that it better fits "medium warship," even though no attempt at a description for the "role" of a "medium warship" appears to have been made here, either. And, indeed, there are multiple other comments suggesting that ship categories in ES are, largely, independent of "role," and instead related to the sizes/masses of the ships. One only needs to look at human medium warships to see this: this is a category that contains both the Corvette and the Bastion. These ships cannot possibly have similar "roles," 

> Also, basically every heavy warship you list are all ones that are called on a regular basis as showcasing why combat in ES is broken, and there is fundamentally no reason to fly anything other than the heaviest ship available.

Relabelling a "heavy warship" to "medium warship" isn't actually going to alleviate the problem you allude to here, if that's your goal. Is your issue just that the ship you want to fly has "heavy warship" written on it but you want it to say "medium warship," even if it still is a heavy warship? You haven't argued that the Albatross is too weak to be a heavy warship, only that it isn't used like one, whatever that means.
So, do you want the Albatross to *be* a medium warship or do you just want it to say "medium warship" on it, even though it isn't? If the former, it should probably have its health, regeneration, and outfit space reduced, if not also its weapon capacity.

> More properly speaking, the Albatross is a "Capital ship" equivalent to the Ibis, and should accordingly be found alongside it.

What does this mean, though? What is a "capital ship" really? When I ask Google, it says:

> a large [warship](https://www.google.com/search?sxsrf=APwXEdcoTDJy8hoDS1bj2xxGpaLRXUp2mQ:1682890892850&q=warship&si=AMnBZoFHF1DJLZWpTBtQDK262RMpPUJwpmu43xR61fNBq5Wb4uatUrEEkCay6jEjqPEZQ-Wvo2sK3oRiKQbSu8Aen42IcR8PFw%3D%3D&expnd=1) such as a [battleship](https://www.google.com/search?sxsrf=APwXEdcoTDJy8hoDS1bj2xxGpaLRXUp2mQ:1682890892850&q=battleship&si=AMnBZoFY6cJe4EcBOpcoqxHCe-Iftnro0UuDmWixUu_lCXwZzhGmqvhce5ie79CoPewQCJuXhPAUdCbGeOZyiYqS2Sonj5kdPQ%3D%3D&expnd=1) or aircraft carrier.

So, the Albatross is a bigger equivalent to the Ibis? And this means it should go in the same category, even though the categories are pretty obviously delineated by size?

In what way is the Albatross "equivalent" to the Ibis anyway? What do they have in common?
The Ibis is a light carrier, the way a player might approach combat with an Ibis is going to be very different to how they might approach combat in an Albatross, which is generally faster, and more durable than its peers (I consider the Shield Beetle and Korath raiders to be its peers, all three are approximately tier 1.5 heavy warships, or approximately tier 1.5 warships with similar capacities) at the expense of weapon capacity and some flexibility in terms of outfitting (since it effectively has a large amount of regeneration permanently installed, leading to the available outfit space being a decent chunk lower than those other ships).

The Ibis doesn't really have any "peers" in the same sense, bar perhaps the Sea Scorpion. Comparing to that, the Ibis has similar total health but more regeneration, but is much slower (it does turn about 22% faster, but the acceleration and speed disparity is so great that the Sea Scorpion could easily reduce thrust and make up the turning gap while still being faster in a straight line, if that were desired. The Ibis does also have much less outfit space than the Sea Scorpion, but some or most of this can also be attributed to the built in regeneration, as with the Albatross. The Ibis does also, however, have two fighter bays and four drone bays.

I have difficulty seeing what similarities would lead to these ships having a similar "role."

> In any case, while holistic change-everything-at-once solutions are nice in theory, I don't think they are practical. Which is why I'm just concerned with getting the Remnant ship classification aligned properly, rather than trying to change everything everywhere. I'm fairly sure that trying to change everything everywhere will never get any kind of widespread agreement.

That's fine, if we don't care about having a consistent game. But, I'm not sure the goal here is to have Remnant content act like a plugin that doesn't really care about how the rest of hte game works as long as that one thing works the way you want it to.

Here's what (the relevant parts of) a shipyard on a Remnant worlds looks like:
image
Why would having the Albatross in the same category as the Ibis be good?

> Hasn't been decided yet, but I rather like the idea of just adding a "Living ship" category that the merganser, peregrine, and any future ships get assigned, and quite obviously if it has both merganser and peregrine in it, it's not being determined by mass or hp totals.

I don't think giving Remnant ships there own set of categories really makes sense. Remnant content is still a part of hte rest of this game, it should be consistent with that. I'm sure you'll tell me that Remnant stuff is "fundamentally different" to the rest of hte game, as you have before, but I've never seen an explanation of that. It doesn't seem that "fundamentally different" to me.

> Also, "capital ship" does not equal "medium warship". It just means "more restricted than regular Remnant ships."

Ah, some elaboration. So, basically, you've made up your own meaning for "capital ship."
As has been mentioned previously by me and some others, the Albatross doesn't really seem like simply a more restricted version of the Ibis. It is a vastly larger, more powerful ship that compares very differently to other similarly sized ships.

**Comment by AvianGeneticist** [comment]
> More properly speaking, the Albatross is a "Capital ship" equivalent to the Ibis, and should accordingly be found alongside it.

I'm not sure where I see this in the Albatross' design, to be perfectly honest. The Ibis is a middleweight strike carrier, quick to act and capable of swarming a target (by ES's standards) with powerful drones and fighters for swift, decisive strikes. (This role can also change to that of an Emergency Response vessel, a well-guarded transport, or a Mining Flagship if fielding Smews instead of Petrels.)

The Albatross, meanwhile, is a large, durable gunship that, while fairly nimble, is more intended for direct capital-on-capital engagements, with no capability for fielding support craft. That, to me, screams "Heavy Warship", since it's designed to be able to go toe-to-toe with another race's large warships.

**Comment by LazerLit** [comment]
I have always thought of the Heavy, Medium, Light, etc. distinctions to be mainly for the player, or _maybe_ from the point-of-view of First Last, rather than how the various species see them. After all, would the Quarg, who can build their own ringworlds, make big distinctions between various sizes of ships (just as an example)?

I like the idea of adding a 'Role' category to ships, and each faction can define their own ships with it. It could even add behaviours too, so, for example, ships with the 'Screen' role would close quickly with enemies and try to soak up missiles. I know I am drifting away from the purpose of this PR, but my point is that maybe this would work better for the Albatross than reclassifying it as a Medium Warship.

**Comment by quyykk** [comment]
At this point it's unlikely that this PR will get merged so we might as well close it
